packages/twenty-front/src/modules/object-record/record-title-cell/hooks/useRecordTitleCell.tsx

```diff
@@ -95,19 +108,29 @@ export const useRecordTitleCell = () => {
         });
         set(isInlineCellInEditModeScopedState(recordTitleCellId), true);
 
-        const fieldDefinitions = objectMetadataItem.fields.map(
-          (fieldMetadataItem, index) =>
-            formatFieldMetadataItemAsColumnDefinition({
-              field: fieldMetadataItem,
-              objectMetadataItem,
-              position: index,
-            }),
-        );
+        const recordIndexFieldDefinitions = snapshot
+          .getLoadable(recordIndexFieldDefinitionsState)
+          .getValue();
 
-        const fieldDefinition = fieldDefinitions.find(
+        let fieldDefinition = recordIndexFieldDefinitions.find(
           (field) => field.metadata.fieldName === fieldName,
         );
 
+        if (!fieldDefinition && isDefined(objectMetadataItem)) {
```

Comment:
Sorry I have a hard time understanding the changes you made in this file, could you please explain what led you to change this?

Reply:
This fixes the regression caused by https://github.com/twentyhq/twenty/pull/13355 which also reported errors in production by sentry

<img width="1030" height="344" alt="image" src="https://github.com/user-attachments/assets/cb4ec797-3ce2-489e-be9d-edcfab36f401" />